django/db/models/sql/query.py

```diff
@@ -12,6 +12,7 @@
 import inspect
 import sys
 import warnings
+import json,ast
```

Comment:
isort [wants](https://djangoci.com/job/isort/9930/console) you to break this import into separate lines.